client/index.html

```diff
@@ -5,44 +5,120 @@
     <meta charset="UTF-8">
```

Comment:
Functionality is really cool, the fact that you can search any key word and results with that keyword appear is very cool. Definitely going to try and implement that in ours.

Reply:
Yeah Kai's done a sick job with it, we're thinking of displaying the results on a new page, but may be more trouble than its worth haha